test/unit/stealth-vault.spec.ts

```diff
@@ -169,9 +167,6 @@ describe('StealthVault', () => {
       then('amount is added to total bonded', async () => {
         expect(await stealthVault.totalBonded()).to.be.equal(bonding);
       });
-      then('updates caller last bond at', async () => {
-        expect(await stealthVault.callerLastBondAt(governor.address)).to.be.gt(initialCallerLastBondAt);
-      });
```

Comment:
should we test for anything else ?

Reply:
don't think so, pretty straightforward change